integration_test/third_party_apps_data/applications/solr/centos_rhel/post

```diff
@@ -0,0 +1,5 @@
+set -e
+
+sudo systemctl restart solr
+
+sudo su - solr -c "/opt/solr/bin/solr create -c otel"
```

Comment:
Is this necessary for the metrics to exist? Can you add a comment describing what this is doing? (maybe `# Create a database called otel so there are some metrics to scrape`?)

Reply:
@quentinmit it is. i can throw a comment in there.

Reply:
@quentinmit done

Reply:
Actually I unresolved this, this also seems like something that's important enough to put in the Markdown docs too (something like "Note: No metrics will be reported if the Solr instance does not contain any cores.")

Reply:
@quentinmit I can add that. But basically Solr is unusable without any cores added, so I don't think a customer will ever set it up without one

Reply:
@quentinmit added a comment in the docs

Reply:
Thanks!